src/firefly/js/ui/panel/CollapsiblePanel.jsx

```diff
@@ -4,77 +4,150 @@
 import {useStoreConnector} from 'firefly/ui/SimpleComponent';
 
 
-export function CollapsiblePanel(props){
-    const isOpenState = useStoreConnector(()=>(getComponentState(props.componentKey)?.isOpen ?? props.isOpen));
-    const newProps = {...props,
-            isOpen : isOpenState,
-            onToggle: (newIsOpen)=>{
-                dispatchComponentStateChange(props.componentKey, {isOpen: newIsOpen});
-                props.onToggle?.(newIsOpen);
-        }};
-
-    return <CollapsiblePanelView {...newProps} />;
-}
-
-
-export function CollapsiblePanelView({header, isOpen=false, headerStyle, contentStyle, style,
-                                         onToggle, children }) {
-    let headerClassName = 'CollapsiblePanel__Header';
-    if (isOpen) headerClassName += ' CollapsiblePanel__Header--is-open';
-    const headerContent = isFunction(header) ? header() : header;
-
-    const contentClassName = 'CollapsiblePanel__Content';
-    const contentHeight = isOpen ? 'initial' : 0;
-    contentStyle = {display: 'block', ...contentStyle, height: contentHeight};
-
+/**
+ * Convenient wrapper to include one CollapsibleGroup and one CollapsibleItem
+ * @param props
+ * @param props.slotProps
+ */
+export function CollapsiblePanel({slotProps, ...props}) {
     return (
-        <div style={{display: 'flex', flexDirection: 'column', ...style}}>
-            <div style={headerStyle} onClick={()=>{onToggle(!isOpen);}} className={headerClassName}>
-                {headerContent}
-            </div>
-            <div style={contentStyle} className={contentClassName}>
-                {children}
-            </div>
-        </div>
+        <CollapsibleGroup {...slotProps?.root}>
+            <CollapsibleItem slotProps={slotProps} {...props} />
+        </CollapsibleGroup>
     );
 }
-
-
-export const FieldGroupCollapsible= memo( (props) => {
-    const {viewProps, fireValueChange}=  useFieldGroupConnector(props);
-    const newProps= {...viewProps,
-            isOpen: Boolean(viewProps.value === 'open'),
-            onToggle: (isOpen) => fireValueChange({ value: isOpen ? 'open' : 'closed' })
-        };
-    return ( <CollapsiblePanelView {...newProps} /> );
-});
-
-CollapsiblePanelView.propTypes = {
-    header: PropTypes.node,
-    isOpen: PropTypes.bool,
-    headerStyle: PropTypes.object,
-    contentStyle: PropTypes.object,
-    style: PropTypes.object,
-    onToggle: PropTypes.func,
-};
-
 CollapsiblePanel.propTypes = {
     componentKey: PropTypes.string.isRequired,
-    ...CollapsiblePanelView.propTypes
+    ...CollapsibleItemView.propTypes
 };
 
+export const FieldGroupCollapsible= memo( ({children, ...props}) => {
+    const {viewProps, fireValueChange}=  useFieldGroupConnector(props);
+    const newProps= {
+        ...pick(viewProps, Object.keys(CollapsibleItemView.propTypes)),  // useFieldGroupConnector is not removing all of its props, causing bad props going into the Tabs
```

Comment:
@robyww I fixed it here and TabPanel but maybe they should be fixed in `useFieldGroupConnector`.

Reply:
field group connector does remove some props but I don't think we can count on it because this is too specific. If CollapsiblePanel is sensitive to extra props then this is probably the best place.